xnmt/lm.py

```diff
@@ -38,7 +38,7 @@ def shared_params(self):
   def get_primary_loss(self):
     return "mle"
 
-  def calc_loss(self, src, trg, loss_calculator):
+  def calc_nll(self, src, trg):
```

Comment:
So MLE always needs a model that calculate nll with two arguments. This is OK so far assuming the supervised model. I guess we will never use MLE for unsupervised model, right?

Reply:
I think we discussed this before when @msperber implemented the LM, and we decided that for now we'd just stick with this design (a simple work-around is to specify the same file for `src` and `trg`) but might revise it later. This is not new in this PR.